opentreemap/treemap/templates/treemap/partials/form_buttons.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+<div class="col-xs-4 text-right">
+    <img class="spinner" src="{{ STATIC_URL }}img/spinner.gif" style="padding: 0 5px 5px 0; display: none;">
```

Comment:
Since there is already a `spinner` class on here, consider moving the inline `padding` styling out into SASS.